psl-cli/src/main/java/org/linqs/psl/cli/Launcher.java

```diff
@@ -112,7 +112,7 @@
 
     private Launcher(CommandLineLoader commandLineLoader) {
         this.options = commandLineLoader.options;
-        this.log = commandLineLoader.log;
+        this.log = LoggerFactory.getLogger(getClass().getName());
```

Comment:
Since we not longer to logging config here, can we make this static (like we do for all other classes)?